### PR TITLE
working documentation examples for sbp package

### PR DESCRIPTION
**Pull request by hmashrique**
The changes include:
 - Fixed the documentation examples to work in the package
 - The functions that are reduntant (**stop.alert, fresh.start**) or dont need examples (**describe.numeric , 
    describe.categorical**) are kept without any examples

### CODE REVIEW

File: R/base-SBP.R [hunk 1]
```diff
@@ -211,12 +211,12 @@ normality.test=function(x)
...
-#' @examples
-#' fresh.start()
+#' @examples #no_need
+#'
 #'
 fresh.start=function()
 {
   rm(list=ls())
-  SBP.code.path="https://raw.githubusercontent.com/stan-pounds/Simple-Biostats-Program/main/"
+  SBP.code.path="https://raw.githubusercontent.com/hmashrique/sbp_associate/main/R/"
```
Comment (aolney) [inline R/base-SBP.R]: Should this path be changed?

Comment (hmashrique) [reply R/base-SBP.R]: i was actually trying to read the files from my repo rather than Stans original one. So the path should be as it was beforehand.

Comment (aolney) [reply R/base-SBP.R]: Great, can you change and push? I'm not sure there is another way in the github workflow

Comment (hmashrique) [reply R/base-SBP.R]: I made the changes. You can check and merge the code now.

**Review by aolney — APPROVED** [review]

**Pull request merged**